setup.py

```diff
@@ -31,82 +21,14 @@
     "requests",
     "requests-toolbelt",
     "rfc3986",
-    "typesystem<0.3",
     "starlette[full]",
+    "typesystem<0.3",
     "uvicorn[standard]",
     "whitenoise",
 ]
 
-
-# https://pypi.python.org/pypi/stdeb/0.8.5#quickstart-2-just-tell-me-the-fastest-way-to-make-a-deb
-class DebCommand(Command):
-    """Support for setup.py deb"""
-
-    description = "Build and publish the .deb package."
-    user_options = []
-
-    @staticmethod
-    def status(s):
-        """Prints things in bold."""
-        print("\033[1m{0}\033[0m".format(s))
-
-    def initialize_options(self):
-        pass
-
-    def finalize_options(self):
-        pass
-
-    def run(self):
-        try:
-            self.status("Removing previous builds…")
-            rmtree(os.path.join(here, "deb_dist"))
-        except FileNotFoundError:
-            pass
-        self.status("Creating debian manifest…")
-        os.system(
-            "python setup.py --command-packages=stdeb.command sdist_dsc -z artful --package3=pipenv --depends3=python3-virtualenv-clone"
-        )
-        self.status("Building .deb…")
-        os.chdir("deb_dist/pipenv-{0}".format(about["__version__"]))
-        os.system("dpkg-buildpackage -rfakeroot -uc -us")
-
-
-class UploadCommand(Command):
```

Comment:
I use this on all my projects.

Reply:
I hear you. However, I think we gradually need to modernize. `setup.py`-based commands are being deprecated by `setuptools` and friends, at some time `setup.py` per se might also become deprecated, and in general, `pyproject.toml`-based project metadata will be here to stay.

In this spirit, project metadata will not be able to include procedural/imperative steps natively. I think it makes sense to accompany modernizations in the Python ecosystem, and will be happy to support you in getting back your loved workflow based on modern utilities.

Effectively, it will just be another incantation command, for example `poe upload` or `poe build-deb`. I think it will be easy to adjust, and you may want to reuse relevant best-of-breed recipes also on your other projects when modernizing them.

If you agree, let's do that on behalf of this project and the next iterations. If you disagree, let's close this PR again.

Reply:
i have mixed feelings, but I'm not opposed. Let's just document the procedures for making a release, etc.

Reply:
thanks again for the help